### ISSUE TITLE
Accessibility requirements

### ISSUE THREAD

**Issue by chrism0dwk** [MikePacey/Lancaster-HEC#1]
The RTD version of the HEC 3.0 doc is really nice -- a clear navigation bar on the left directs users to the topic they need quickly, and you can navigate around the documentation quickly and effectively.  Answers has been slated by current HEC users and the wider LU community alike, so I think it's worth making a point here!

For ISS to be swayed on this, we will need to show that the site comes up to accessibility standards.  The only obvious thing I can see as of now is the lack of Alt-text in the images: could this be added?

**Comment by MikePacey** [comment]
The only image I'm seeing in the docs is the diagram of the HEC in the howitworks section. The original markup contains an alt tag and I can see from the published html source that it's there in the img tag. Was there some other image you were referring to?

**Comment by chrism0dwk** [comment]
Hmm, interesting.  Mouse over tooltip not showing in Chrome for Linux.   Odd.

**Comment by MikePacey** [comment]
I believe that's a bit of functionality from old versions of IE which isn't supported any longer. These days its the title tag that shows on mouse over, not the alt tag. The alt tag gets displayed when the image can't be displayed (e.g. with a screenreader).

**Comment by chrism0dwk** [comment]
Nice!
